how-tos/releaseduty_overview.md

```diff
@@ -0,0 +1,21 @@
+# ReleaseDuty Statement of Work / Overview
```

Comment:
Nit: There is a Buildduty "manifesto". I'm thinking we could keep the consistency and name this `releaseduty_manifesto` as filename. The title is very good though.

Reply:
'Statement of Work' has a specific meaning in project management, I'm given to understand that's what was asked for, but if it's not, we can change it.

https://en.wikipedia.org/wiki/Statement_of_work

Reply:
Yes, absolutely. I totally agree the title is good. I was just referring to renaming `releasesduty_overview` to `releaseduty_manifesto` 👍 Fine either way, though.